.github/workflows/rust-standalone.yml

```diff
@@ -9,30 +9,35 @@ on:
     branches:
       - master
       - "rust/*"
+      - "ci/update-rust-standalone"
 
 jobs:
-  build_ubuntu_2204:
-    name: Ubuntu-22.04
-    runs-on: ubuntu-22.04
+  build_ubuntu:
+    name: Ubuntu
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
+    container:
+      image: wasmedge/wasmedge:ubuntu-build-clang
```

Comment:
If you are using our internal docker image, then it will always be ubuntu 20.04. The matrix is totally useless.

Reply:
Ok. I'll refactor this part to support both versions.